services/frontend-service/pkg/cmd/server.go

```diff
@@ -125,14 +125,21 @@ func RunServer() {
 	}
 }
 
+func stripSecrets(config config.ServerConfig) config.ServerConfig {
+	if config.DexClientSecret != "" {
+		config.DexClientSecret = "**********"
+	}
+	return config
+}
+
 func runServer(ctx context.Context) error {
 	err := envconfig.Process("kuberpult", &c)
 
 	if err != nil {
 		logger.FromContext(ctx).Error("config.parse", zap.Error(err))
 		return err
 	}
-	logger.FromContext(ctx).Warn(fmt.Sprintf("config: \n%v", c))
+	logger.FromContext(ctx).Warn(fmt.Sprintf("config: \n%v", stripSecrets(c)))
```

Comment:
I think we can just remove this log line entirely.

Reply:
alternatively: Just log what it's interesting.
I would say that all boolean values are kinda interesting and the rest can be skipped most likely.

Reply:
So you propose setting up e.g. a custom `func (config config.ServerConfig) String() string` function where we define what should be logged?

From my perspective the log helped in the past to verify the setup is correct, so I would not drop it. However, manually adding new fields to it in the future is also a little cumbersome. 

So please decide, and I'll implement accordingly :laughing: